commands/plugin-uninstall.js

```diff
@@ -0,0 +1,114 @@
+'use strict';
+
+const spawn = require('child-process-ext/spawn');
+const fsp = require('fs').promises;
+const fse = require('fs-extra');
+const path = require('path');
+const _ = require('lodash');
+const isPlainObject = require('type/plain-object/is');
+const yaml = require('js-yaml');
+const cloudformationSchema = require('@serverless/utils/cloudformation-schema');
+const log = require('@serverless/utils/log');
+const yamlAstParser = require('../lib/utils/yamlAstParser');
+const npmCommandDeferred = require('../lib/utils/npm-command-deferred');
+const {
+  getPluginInfo,
+  getServerlessFilePath,
+  validate,
+} = require('../lib/commands/plugin-management');
+
+module.exports = async ({ configuration, serviceDir, configurationFilename, options }) => {
+  validate({ serviceDir });
+
+  const pluginInfo = getPluginInfo(options.name);
+  const pluginName = pluginInfo.name;
+  const pluginVersion = pluginInfo.version || 'latest';
```

Comment:
In case of `uninstall` I believe _version_ is irrelevant so we can skip that part